### ISSUE TITLE
Handle Status Bar Sizing?

### ISSUE THREAD

**Issue by joecaraccio** [wix/react-native-navigation#2999]
This is particular to iOS, but is there a way to globally account for the status bar offset.

I don't use the Nav Bar because I can't seem to completely override it visually. Is there a way to add some sort of global padding to the top of the view (so you offset for the status bar on iOS)

**Comment by joecaraccio** [comment]
@guys sorry, what's HOC?

**Comment by guyca** [comment]
https://medium.com/@bosung90/use-higher-order-component-in-react-native-df44e634e860

**Comment by joecaraccio** [comment]
@guyca Have you done something like that with React Native Navigation? I would know how to do that if I weren't using RNN, but with RNN you start components with "StartTabBased" etc.

**Comment by guyca** [comment]
Not exactly, you declare the application's layout and style in `startTabBasedApp`, but the actual components are provided by the generator function you pass as argument when calling `registerComponent`. Here's a nice [example](https://github.com/wix/react-native-navigation/issues/2525)
